src/parquet/column_reader.cc

```diff
@@ -269,4 +167,36 @@ bool ColumnReader::ReadNewPage() {
   return true;
 }
 
+std::shared_ptr<ColumnReader> make_column_reader(const parquet::ColumnMetaData* metadata,
```

Comment:
Done. What do you think about making this a static method like `ColumnReader::Make` ?